### ISSUE TITLE
Epic: Manage CodeQL deployment

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#5005]
### Overview
We require issues to support resolution of current open CodeQL alerts as well as automation for creation of issues for new CodeQL alerts.  We also need to manage CodeQL extraction errors and consider implementing CodeQL for VS Code extension.  

### Dependencies
- [ ] #5242
- [ ] #6463
- [ ] #5159 
- [x] #5059
- [x] #5234
- [ ] #6548
- [x] #5180
- [x] #6691

### Action Items
- [x] View the code scanning page[^1] for hackforla/website, and understand the elements of CodeQL code scanning[^2]
- [x] Create issue template #5242
- [x] Create a sample issue - #5642
- [x] Create a  level 1 issue making issue with a list of required issues - see #5159
- [x] Create an issue for creation of a wiki page "How to manage CodeQL alerts" - #6463
- [ ] fix link from Action Item to alert
- [ ] set role label based on path of affected code file
   - [ ] if path starts with `.github` or `github-actions` => apply `role: back end/devOps
   - [ ] otherwise apply `role: front end` label
- [ ] determine if the code file is in production
   - [ ] for pages, check YAML front matter for `published: `
   - [ ] for layouts, check `_config.yml`

### Resources/Instructions
[^1]: [Code Scanning results for hackforla/website](https://github.com/hackforla/website/security/code-scanning) 
[^2]: [About code scanning with CodeQL](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-with-codeql)
[^3]: spreadsheet [HfLA website: CodeQL scan alerts  (issue #5060)](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=0)
- This Epic was created from #4963

**Comment by ExperimentsInHonesty** [comment]
- @roslynwythe There was a dependency on this for #5060 to be done first, but I think the issue template for auditing has to be made first, so that the issues in 5060 can be made.  So I have fixed it.  If you agree, re add the ready for prioritization label and I will take care of it.  If you don't agree, please explain what I might be misunderstanding.

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty The requirements for managing the alerts is actually quite similar to the requirements for the img tag refactoring Roll Out:  a number of auditing/analysis issues, followed by a set of code-changing issues, with the use of a tracking spreadsheet.   So I'm wondering if I this issue should be a Roll Out Plan with Issue Making Level 3, and the proposed issue-making issues should be Epics with Issue Making Level 2.   The Roll Out plan might also include other issues related to CodeQL alerts, such as issues around notifications,  monitoring, and documentation.   Please advise.  

Regarding the depedencies:  In Jessica's Roll Out plan, she listed the lower-level issues (Epics) as dependencies.  So it seemed to me that for Roll Out plans and Epics, the meaning of dependencies is a bit turned around, and they should be interpreted to mean that the higher-level issues must remain open until the lower-level issues are completed.  You have my sincere apologies for the confusion...

One more thing, I have already started work on the spreadsheet and template, so I was hoping to self-assign this issue.

**Comment by roslynwythe** [comment]
> * @roslynwythe There was a dependency on this for [Create Issues for Analysis of CodeQL alerts #5060](https://github.com/hackforla/website/issues/5060) to be done first, but I think the issue template for auditing has to be made first, so that the issues in 5060 can be made.  So I have fixed it.  If you agree, re add the ready for prioritization label and I will take care of it.  If you don't agree, please explain what I might be misunderstanding.

Yes, thank you.

**Comment by roslynwythe** [comment]
### Issue 1 (Analysis) Template Draft 

### (Title) Analysis for CodeQL query `[QUERY NAME]`

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to analyze the query `[QUERY NAME]` which appears in the CodeQL code scan results[^2] and we need to make recommendations about the disposition of each specific alert found from that query.  Analysis and Recommendations will be recorded in the spreadsheet: HfLA website: CodeQL scan alerts[^1]

### Action Items
- [ ] Open the spreadsheet HfLA website: CodeQL scan alerts[^1] (`CodeQL query types` sheet) and locate the row in which the `Query` column contains `[QUERY NAME]`
- [ ] From the **code scanning results page**[^2] browse to an alert with query type `[QUERY NAME]`.  Copy the query, severity level, description, and recommendation text from the webpage into a new row in the `CodeQL query types` sheet. 
- [ ] Observe the query tags listed on the right column of the webpage.  If the tag `security` is listed, check the "Security Alert?" column of the spreadsheet.   
- [ ] Add your analysis and recommendation
   - [ ] In the "recommended action" column, select from the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, `change code`, or `varies by instance`
   - [ ] DO NOT DISMISS ANY ALERTS in the code scanning results page[^2]
- [ ] Populate a row in the `Alert Instances` sheet for each alert instance in the **code scanning results page**[^2]; Complete the `query`, `link to alert page` and `file` columns  
  - [ ] If the recommendation is `fix code`, populate these columns:
    - [ ]  `Line of current code`
    - [ ]  `Recommended line of code`
    - [ ] `web page` - URL of the affected web page, for testing purposes
    - [ ] `test procedure`- describe the process which can be used after the code fix has been made, to check that the change has not broken anything.    
- [ ] After all instances have been recorded in the sheet, add the `ready for dev lead` label to this issue and move it to the `Questions/In Review column`

### For merge team/dev lead
- [ ] To review this issue:
   - [ ] Locate the row in the `CodeQL query types` sheet.  All columns should be populated. 
   - [ ] Locate the rows in the`Alert Instances`.  There should be a row for each instance of the alert in the **code scanning results page**[^2].  For each row:
     - [ ] If the recommendation is `change code`, all columns must be completed with the exception of `Link to Issue`
     - [ ] If the recommendation is to dismiss, none of the remaining columns are required (as indicated on the sheet)
- [ ] Review recommendations.  If you agree with the recommendation to dismiss the alert, dismiss the alert.  
- [ ] Remove the `ready for dev lead` label and close the issue
- [ ] If new issues are required to make code fixes, put the issue in "New Issue Approval" with "ready for dev lead" label
- [ ] Check off the Issue in #5060
  
### Resources/Instructions
[^1]: spreadsheet: [HfLA website: CodeQL scan alerts - issue #5060](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043)
[^2]:[Code scanning results page](https://github.com/hackforla/website/security/code-scanning)

- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is tracked in the spreadsheet [HfLA website: CodeQL scan alerts (issue #5060)](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043) and #5060

**Comment by roslynwythe** [comment]
> * @roslynwythe There was a dependency on this for [Create Issues for Analysis of CodeQL alerts #5060](https://github.com/hackforla/website/issues/5060) to be done first, but I think the issue template for auditing has to be made first, so that the issues in 5060 can be made.  So I have fixed it.  If you agree, re add the ready for prioritization label and I will take care of it.  If you don't agree, please explain what I might be misunderstanding.

- @ExperimentsInHonesty I understand that the template must be made first before the issues can be made, but if you list 
this Epic as a dependency on #5060.  doesn't that imply that this Epic must be closed before work can be begin on #5060?  This Epic encompasses both the auditing and the code changes resulting from CodeQL alerts, so I was expecting that it would remain open until all those issues are created or even until they are completed.    

- When I setup the dependencies the other way around (listing #5060 as a dependency on this issue), I was attempting to model this issue after other Epics and Roll Out Plans (for example #4362), where I noticed that the lower-level issue-making issues were listed as dependencies on the higher-level issues.  At first that logic seemed backward but now I understand it to mean that the higher level issue must remain open until the all the lower level issues are closed.  

But I'm ok however we setup the dependencies.  

- Also, there are some other requirements related to CodeQL:
  - Issues for managing secret scanning
  - Issues creating email notifications for secret scanning and CodeQL alerts (I believe these may be require GHAs) - #5059
  - ~~An Issue to clarify policy/documentation for reviewing alerts as part of the Pull Request Review process~~

and I'm not sure, should those requirements be incorporated into this Epic?

I would like to remain assigned to this issue. 

Thank you

**Comment by roslynwythe** [comment]
### Issue 2 (Code Fix) Template Draft 

### (Title) Code Change for resolve CodeQL alert

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to make a code change in order to resolve a CodeQL alert [LINK TO ALERT PAGE]

### Action Items
- [ ] Open the file `[FILE PATH]` in your IDE
- [ ] Replace the line
```
[REPLACE WITH EXISTING LINE OF CODE]
```
with
```
[REPLACE WITH DESIRED LINE OF CODE]
```
- [ ] Using docker, browse to [WEB PAGE].  [TEST PROCEDURE]

### For merge team/dev lead
  - [ ] Check off the Issue in #5159
  
### Resources/Instructions
- [Code scanning results page](https://github.com/hackforla/website/security/code-scanning)
- This issue is part of #5159

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty Question: in the case in which the recommendation is to dismiss an alert, how is that managed?  Options:
- [ ] merge team member dismisses the alert after reviewing/closing the issue
- [ ] we create a new issue for dismissing the alert.

**Comment by roslynwythe** [comment]
Issue 1 (analysis) template
```
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to analyze the query type `Potentially unsafe external link` which appears in the CodeQL code scan results on the hackforla/website repository, and to make recommendations about the disposition of each specific alert found with that query.  Analysis and Recommendations will be recorded in the spreadsheet: **HfLA website: CodeQL scan alerts[^1]

### Action Items
- [x] Open the spreadsheet HfLA website: CodeQL scan alerts[^1]
- [x] From the **code scanning results page**[^2] browse to an alert with query type `Potentially unsafe external link`.  Copy the query, severity level, description, and recommendation text from the webpage into a new row in the spreadsheet. 
- [x] Observe the query tags listed on the left column of the webpage.  If the tag `security` is listed, check the "Security Alert?" column of the spreadsheet.  
- [x] Add your analysis and recommendation
   - [x] In the "recommended action" column, select from the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, `fix code`, or `varies by instance`
   - [x] If the recommendation is `fix code`, describe the code change in the `recommended code change` column
- [x] Populate a row in the `Specific Alert Instances` sheet for each alert instance, by completing the `query`, `alert #` and `file` columns  
- [x] If the recommndation is `fix code`, populate these columns:
  - [x]  `original code`
  - [x]  `recommended code change`
  - [x] `web page` - URL of the affected web page, for testing purposes
  - [x] `test procedure`- describe the process which can be used after the code fix has been made, to check that the change has not broken anything.    
- [x] After all instances have been recorded in the sheet, add the `ready for dev lead` label to this issue and move it to the `Questions/In Review column`

### For merge team/dev lead
- [x] After reviewing the issue
   - [x] If the recommendation is to dismiss one or more alerts, go to the **code scanning results page**[^2] page and dismiss each specific alert using the specified message. 
   - [x] Remove the `ready for dev lead` label and close the issue
   - [x] Check off the Issue in #5060
  
### Resources/Instructions
[^1]: spreadsheet: [HfLA website: CodeQL scan alerts](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043)
[^2]:[Code scanning results page](https://github.com/hackforla/website/security/code-scanning)

- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is tracked in the spreadsheet [HfLA website: CodeQL scan alerts](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043) and #5060
```

**Comment by roslynwythe** [comment]
Template for Analysis for CodeQL alerts detected within Pull Requests

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to analyze the CodeQL alert of type `[QUERY NAME]` detected in the file [FILE] in Pull Request [PULL REQUEST] and we need to make recommendations about the disposition of the alert.  Analysis and Recommendations will be recorded in the spreadsheet: HfLA website: CodeQL scan alerts[^1]

### Action Items
- [ ] Browse to the Pull Request #[PULL REQUEST NUMBER] and locate the code annotation describing the alert.  Browse to the alert details page (link labelled "show more details") 
- [ ] Open the spreadsheet HfLA website: CodeQL scan alerts[^1] (`CodeQL query types` sheet) and locate any rows in which the `Query` column contains `[QUERY NAME]`.  If a row exists, read and understand the previous handling of similar alerts.  If a row does not exist, create a new row, then copy the query, severity level, description, and recommendation text from the webpage into a new row in the `CodeQL query types` sheet. 
- [ ] Observe the query tags listed on the right column of the webpage.  If the tag `security` is listed, check the "Security Alert?" column of the spreadsheet.   
- [ ] Add your analysis and recommendation
   - [ ] In the "recommended action" column, ?????? select from the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, `change code`, or `varies by instance`
   - [ ] DO NOT DISMISS ANY ALERTS in the code scanning results page[^2]
- [ ] Populate a row in the `Alert Instances` sheet; Complete the `query`, `link to alert page` and `file` columns  
  - [ ] If the recommendation is `fix code`, populate these columns:
    - [ ]  `Line of current code`
    - [ ]  `Recommended line of code`
    - [ ] `web page` - URL of the affected web page, for testing purposes
    - [ ] `test procedure`- describe the process which can be used after the code fix has been made, to check that the change has not broken anything.    
- [ ] After all instances have been recorded in the sheet, add the `ready for dev lead` label to this issue and move it to the `Questions/In Review column`

### For merge team/dev lead
- [ ] To review this issue:
   - [ ] Locate the row in the `CodeQL query types` sheet.  All columns should be populated. 
   - [ ] Locate the rows in the`Alert Instances`.  There should be a row for each instance of the alert in the **code scanning results page**[^2].  For each row:
     - [ ] If the recommendation is `change code`, all columns must be completed with the exception of `Link to Issue`
     - [ ] If the recommendation is to dismiss, none of the remaining columns are required (as indicated on the sheet)
- [ ] Review recommendations.  If you agree with the recommendation to dismiss the alert, dismiss the alert.  
- [ ] Remove the `ready for dev lead` label and close the issue
- [ ] If new issues are required to make code fixes, put the issue in "New Issue Approval" with "ready for dev lead" label
- [ ] Check off the Issue in #5060
  
### Resources/Instructions
[^1]: spreadsheet: [HfLA website: CodeQL scan alerts - issue #5060](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043)
[^2]:[Code scanning results page](https://github.com/hackforla/website/security/code-scanning)

- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is tracked in the spreadsheet [HfLA website: CodeQL scan alerts (issue #5060)](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043) and #5060

**Comment by roslynwythe** [comment]
### (Title) Analysis for CodeQL alert `[ALERT #]`

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to analyze [CodeQL query alert {INSERT ALERT #}](https://github.com/hackforla/website/security/code-scanning/{INSERT-ALERT-#}) and to recommend appropriate disposition of the alert. Options include dismissal of the alert or code fix;  recommendations will be recorded in the spreadsheet: HfLA website: CodeQL scan alerts[^3]

### Action Items
- [ ] Open the spreadsheet HfLA website: CodeQL scan alerts[^1] and locate the row for `alert #` `[ALERT #]`
- [ ] Browse to the alert url[^2].  Note the `severity level` from the webpage and set the `severity level` column of the spreadsheet accordingly.
- [ ] Observe the query tags listed on the right column of the webpage.  If the tag `security` is listed, check the "Security Alert?" column of the spreadsheet.   
- [ ] Add your analysis and recommendation
   - [ ] In the "recommended action" column, select from the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, or `change code`
   - [ ] DO NOT DISMISS ANY ALERTS.  Dismissal of alerts should be done by merge team only after review of this issue. 
- [ ] Complete the `query`, `link to alert page` and `file` columns  
  - [ ] If the recommendation is `fix code`, populate these columns:
    - [ ]  `Line of current code`
    - [ ]  `Recommended line of code`
    - [ ] `web page` - URL of the affected web page, for testing purposes
    - [ ] `test procedure`- describe the process which can be used after the code fix has been made, to check that the change has not broken anything.    
- [ ] Add the `ready for dev lead` label to this issue and move it to the `Questions/In Review column`

### For merge team/dev lead
- [ ] To review this issue:
   - [ ] Locate the row in the `CodeQL query types` sheet.  
     - [ ] If the recommendation is `change code`, all columns must be completed with the exception of `Link to Issue`
     - [ ] If the recommendation is to dismiss, none of the remaining columns are required (as indicated on the sheet)
- [ ] Review recommendations.  If you agree with the recommendation to dismiss the alert, dismiss the alert.  
- [ ] Remove the `ready for dev lead` label and close the issue
- [ ] If new issues are required to make code fixes, put the issue in "New Issue Approval" with "ready for dev lead" label
- [ ] Check off the Issue in #5060
  
### Resources/Instructions

[^1]: spreadsheet: [HfLA website: CodeQL scan alerts - issue #5060](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043)
[^2]:[CodeQL query alert {INSERT ALERT #}](https://github.com/hackforla/website/security/code-scanning/{INSERT-ALERT-#})


- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is tracked in the spreadsheet [HfLA website: CodeQL scan alerts (issue #5060)](https://docs.google.com/spreadsheets/d/1B3R-fI8OW0LcYuwZICQZ2fB8sjlE3VsfyGIXoReNBIs/edit#gid=193401043) and #5060

**Comment by roslynwythe** [comment]
Revised issue template - note that there are 3 instances of 'INSERT-ALERTID' below

Title:  Resolve CodeQL Alert INSERT-ALERTID: Recommend dismissal or update code

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
As developers. we need to analyze [CodeQL query alert INSERT-ALERTID](https://github.com/hackforla/website/security/code-scanning/INSERT-ALERTID) and to either recommend dismissal of the alert or update the code to resolve the alert.    

### Action Items
- [ ] DO NOT DISMISS ANY ALERTS.  Dismissal of alerts should be done by dev leads only after review of the recommendation
- [ ] Browse to the link in the next Action Item and read the contents.  Click "See More" to view Recommendations, Examples and References.  
- [ ] https://github.com/hackforla/website/security/code-scanning/INSERT-ALERTID 
- [ ] In a comment in this issue, add your analysis and recommendations with details.  The recommendation can be one of the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, or `update code`.  An example of a 'false positive' is a report of a JavaScript syntax error that is caused by markdown or liquid symbols such as `---` or `{%`.  
- [ ] If the recommendation is to update code:
   - [ ] create an issue branch and proceed with the code update
   - [ ] Use docker to test locally, ensuring that there are no changes to any affected webpage(s)
   - [ ] proceed with pull request in the usual manner 
- [ ] If the recommendation is to dismiss, describe your reason for dismissal in the comment, then move the issue to `Questions/In Review` and apply the label `ready for dev lead`.  


### For merge team/dev lead
- [ ] If recommendation to dismiss is approved, dismiss the alert with a comment, then close the issue as completed.
- [ ] When this issue is closed please check off the dependency (under "Analysis Issues") in #5005.  If all analysis issues are closed, close #5005 as completed.  
  
### Resources/Instructions
- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- This issue is part of #5005

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 13, 2024 at 12:14 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 27, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, September 3, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, September 10, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, September 17, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 8, 2024 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 15, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 22, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, October 29, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, November 4, 2024 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, November 11, 2024 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, November 18, 2024 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, November 25, 2024 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, December 30, 2024 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 6, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 13, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 20, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 27, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 3, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 10, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 17, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, February 24, 2025 at 11:05 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, March 3, 2025 at 11:04 PM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, March 11, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, March 18, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, March 25, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, April 1, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, April 8, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, April 15, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, April 22, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, April 29, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, May 6, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, May 13, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, May 19, 2025 at 11:09 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, May 20, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, May 27, 2025 at 12:05 AM PST.</sub>

**Comment by HackforLABot** [comment]
@roslynwythe

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, June 3, 2025 at 12:05 AM PST.</sub>
